Rakefile

```diff
@@ -16,4 +16,7 @@ rescue LoadError
   end
 end
 
-task default: %i[spec rubocop]
+# remove default rspec task
+task(:default).clear
```

Comment:
I've not seen this before.  Is it necessary in addition to the line below?

Reply:
@ndushay yeah, else rspec hijacks the default rake task. have had to do this on a buuuuuuunch of projects.